package.json

```diff
@@ -109,7 +109,8 @@
 			},
 			{
 				"command": "v.playground",
-				"title": "V: Upload and share current code to V playground"
+				"title": "V: Open current file on DevBits V playground",
+				"enablement": "editorLangId == v"
```

Comment:
I switched to `when` property in another PR, so it can be removed in favor of `when`.

Reply:
Yeah... I notice that. will push later.